src/bin2pat/bin2pat.cpp

```diff
@@ -81,6 +85,30 @@ void processArgs(
 				return;
 			}
 		}
+		else if (args[i] == "-j" || args[i] == "--json") {
+			Json::Value jsonIn;
+			Json::Reader reader;
+			std::ifstream jsonFile(args[++i], std::ifstream::binary);
```

Comment:
What if `-j` or `--json` is the very last argument? Wouldn't this crash when trying to access `++i`? Shouldn't we also check that there is one more argument after `-j`/`--json`?